### PR TITLE
Add team page

### PR DESCRIPTION
**Pull request by anneschuth**
# Description

Adds a team page.

image

### CODE REVIEW

File: docs/About/team.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+-   ## Robbert Bos
+
+    Product Owner
+
+    ----
+
+    ![Robbert Bos](/img/robbert_bos.jpeg){: .round }
+
+    Robbert is....
```
Comment (anneschuth) [inline docs/About/team.md:13]: @robbertbos please add

File: docs/About/team.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+-   ## Berry den Hartog
+
+    Engineer
+
+    ----
+
+    ![Berry den Hartog](/img/berry.jpeg){: .round }
+
+    Berry is ...
```
Comment (anneschuth) [inline docs/About/team.md:42]: @berrydenhartog please add

File: docs/About/team.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+-   ## Willy Tadema
+
+    AI Ethics Lead
+
+    ----
+
+    ![Lucas Willy Tadema](/img/willy.png){: .round }
+
+    Willy is
```
Comment (anneschuth) [inline docs/About/team.md:73]: @FrieseWoudloper please add

File: docs/About/team.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+-   ## Robbert Uittenbroek
+
+    Engineer
+
+    ----
+
+    ![Robbert Uittenbroek](/img/robbert_uittenbroek.jpeg){: .round }
+
+    Robbert is...
```
Comment (anneschuth) [inline docs/About/team.md:87]: @uittenbroekrobbert please add

File: docs/About/team.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+-   ## Laurens Weijs
+
+    Engineer
+
+    ----
+
+    ![Laurens Weijs](/img/laurens.jpeg){: .round }
+
+    Laurens is ...
```
Comment (anneschuth) [inline docs/About/team.md:101]: @laurensWe please add

File: docs/About/team.md (hunk 4 above)
Comment (uittenbroekrobbert) [inline docs/About/team.md:87]:
```suggestion
Robbert is a highly enthusiastic full-stack engineer with a Bachelor's degree in Computer Science from the Hanze University of Applied Sciences in Groningen. He is passionate about building secure, compliant, and ethical solutions, and thrives in collaborative environments. Robbert is eager to leverage his skills and knowledge to help shape and propel the future of IT within the government.
```

File: docs/About/team.md (hunk 2 above)
Comment (berrydenhartog) [reply docs/About/team.md]: Done

File: .pre-commit-config.yaml [hunk 6]
```diff
@@ -17,3 +17,4 @@ repos:
     rev: v0.39.0
     hooks:
       - id: markdownlint
+        args: [ --disable, MD030, MD033 ]
```
Comment (berrydenhartog) [inline .pre-commit-config.yaml:20]: i would prefer not disabling rules for all documents but use a disable rule line (one of these examples)
```
<!-- markdownlint-disable MD030 MD033 -->
<!-- markdownlint-disable-next-line MD030 MD033 -->
```

Comment (anneschuth) [reply .pre-commit-config.yaml:20]: Wasn't aware that was possible, thanks,  will do!

File: docs/About/team.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+-   ## Willy Tadema
+
+    AI Ethics Lead
+
+    ----
+
+    ![Willy Tadema](../img/willy.png){: .round }
+
+    Willy specializes in data science, ethics-by-design, XAI, fairness in machine learning, ML model risk management,
+    ML architecture, and MLOps. He's also skilled in R development, data integration, quality management, and geographic
```
Comment (robbertbos) [inline docs/About/team.md:82]:
```suggestion
    ML architecture, and MLOps. She's also skilled in R development, data integration, quality management, and geographic
```

**Review by robbertbos — CHANGES_REQUESTED** [review]

**Review by robbertbos — APPROVED** [review]

**Pull request merged**